### ISSUE TITLE
Certain names for types break deployment

### ISSUE THREAD

**Issue by mlukaszczyk** [prisma/prisma#2659]
**Describe the bug**
The pluralization library that prisma uses has a set of "uncountable" words. This means that the plural form is the same as the singular form. 

Example: "News" will result in "News" after the pluralization. This results in the main entrypoint for single and multiple queries (I guess other parts of the schema are affected as well) to get the same name, which leads to a failed deployment.

This is the place in the pluralization lib: https://github.com/atteo/evo-inflector/blob/64ab921bbdeb797e6aa6469eb7a53320b505221a/src/main/java/org/atteo/evo/inflector/English.java#L133

```
uncountable(new String[] {
			// 2. Handle words that do not inflect in the plural (such as fish, travois, chassis, nationalities ending
			// endings
			"fish", "ois", "sheep", "deer", "pox", "itis",

			// words
			"bison", "flounder", "pliers", "bream",
			"gallows", "proceedings", "breeches", "graffiti", "rabies",
			"britches", "headquarters", "salmon", "carp", "herpes",
			"scissors", "chassis", "high-jinks", "sea-bass", "clippers",
			"homework", "series", "cod", "innings", "shears",
			"contretemps", "jackanapes", "species", "corps", "mackerel",
			"swine", "debris", "measles", "trout", "diabetes", "mews",
			"tuna", "djinn", "mumps", "whiting", "eland", "news",
			"wildebeest", "elk", "pincers", "sugar" });
```

Most of these words, like "britches" will probably never have the honor of ending up as a GraphQL type, except "News" of course. 

This wasn't an issue in graphcool before, because of the "all..." prefix.

**Comment by dpetrick** [comment]
Thank you for the report, we'll look into it.

**Comment by divyenduz** [comment]
Currently, this lets us deploy to Prisma server, however, the service never comes up.

**Comment by mavilein** [comment]
Fixed in `1.19.2` which will be available in a few minutes.
